### PR TITLE
chore: Add kn-plugin-migration to peribolos configuration

### PR DESCRIPTION
**Pull request by rhuss**
and sorting the kn plugins in alphabetical order.

### CODE REVIEW

File: peribolos/knative-sandbox.yaml [hunk 1]
```diff
@@ -362,6 +362,18 @@ orgs:
         description: Kn plugin for managing a Kubernetes based Knative installation.
         has_projects: true
         has_wiki: false
+      kn-plugin-diag:
+        allow_merge_commit: false
+        allow_rebase_merge: false
+        description: Kn plugin for exposing detailed information for different layer of knative objects for diagnose.
```
Comment (vaikas) [inline peribolos/knative-sandbox.yaml:368]: nit s/layer/layers/?

Comment (markusthoemmes) [reply peribolos/knative-sandbox.yaml:368]: ack

Comment (rhuss) [reply peribolos/knative-sandbox.yaml]: fixed. Thx !

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

**Comment by vaikas** [comment]
/lgtm
/approve

Please check this after it gets merged to make sure there are no issues:
https://prow.knative.dev/?job=post-knative-sandbox-peribolos

**Comment by vaikas** [comment]
@rhuss FYI: https://github.com/knative/test-infra/issues/2652

**Comment by vaikas** [comment]
@rhuss checks out:
https://prow.knative.dev/view/gs/knative-prow/logs/post-knative-sandbox-peribolos/1358818851202011136

**Pull request merged**